src/pages/HomePage.jsx

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import Input from '../components/Input';
+import Chat from '../components/Chat';
+import $ from 'jquery';
+
+export default function CHRAT({messages, addNewMessage}) {
+  function goToScrollDown() {
+    $('#mainChat').scrollTop($('#mainChat')[0].scrollHeight);
+  }
```

Comment:
You should not use jquery or vanilla JS direct DOM access functions within a React project. If React does not control the DOM completely this can lead to unexpected behavior. Have a look at `useRef()` if you need to access DOM elements directly.

Reply:
jah the problem was that {window.scrollTo(0, document.body.scrollHeight)} did not work and i just tried several things